packages/amplify/amplify_flutter/lib/src/hybrid_impl.dart

```diff
@@ -74,7 +91,7 @@ class AmplifyHybridImpl extends AmplifyClassImpl {
           plugin.streamController.stream,
         );
       } else if (plugin is APIPluginInterface) {
-        await API.addPlugin(plugin);
+        await API.addPlugin(plugin, authProviderRepo: authProviderRepo);
```

Comment:
It's only required if not adding it would cause the line to wrap. In this case, it's purely a matter of personal preference, although I can see the argument for having them everywhere there are 2+ parameters. Hmm, it just becomes a bit cluttered if it's used so militantly. For example, in this case it may look alright, but for something like `sum(1, 2)`, adding a trailing comma would be clearly a waste of space turning 1 line into 4.